metale_amorficzne/pages/02_Data.py

```diff
@@ -0,0 +1,4 @@
+import streamlit as st
+from metale_amorficzne.streamlit_utils import show_markdown
+
+show_markdown(__file__, unsafe_allow_html=True)
```

Comment:
Here you can skip `unsafe_allow_html`, as no explicit HTML seems to be included in `01_Data.md`.

Reply:
ok